src/constants.rs

```diff
@@ -1,5 +1,3 @@
 pub const OIDC_SESSION_KEY: &str = "oidc_context";
 pub const SESSION_COOKIE_NAME: &str = "opendid";
 pub const ID_TOKEN_VARIABLE_NAME: &str = "ID_TOKEN";
-pub const RESPONSE_TYPE_SESSION_KEY: &str = "response_type";
-pub const REDIRECT_URI_SESSION_KEY: &str = "redirect_uri";
```

Comment:
there was no need for these since the entire Authorize Parameters are stored in a session.